spec/controllers/graphql/queries/freaks_spec.rb

```diff
@@ -13,8 +13,17 @@ module Graphql
     end
 
     before do
-      create(:freak, first_name: 'Ion')
-      create(:freak, first_name: 'Gheorghe')
+      create(:freak)
+    end
+
+    context 'with a full_time freak' do
+      it 'returns a full_time freak' do
```

Comment:
Where's the full-time freak created within this context, or the full-time norm associated to the existing freak?

Reply:
@gabrielgiroe1 I got no answer to the question nor I see any significant changes made to let the code answer the question.
![image](https://user-images.githubusercontent.com/24190790/130752401-a84c7179-42e5-44bf-b01e-bca7550e1aba.png)